<!DOCTYPE html><html><head><title>Getting started with CRISPR experiments: Part I</title></head><body>
<h1>Getting started with CRISPR experiments: Part I</h1><p><a href="http://genomics.quiltdata.com/2016/01/26/getting-started-with-crispr-experiments-part-i/" target="_new">Original URL</a></p>
<p><blockquote>CRISPR as molecular debugging CRISPR technology is transforming how we manipulate genes to understand their function. To explain CRISPR&#x2019;s emerging role in genetic engineering, let&#x2019;s use&hellip;</blockquote></p>
<div><div class="entry-content">
		<h2><span>CRISPR as molecular debugging</span></h2>
<p><a href="https://goo.gl/9RVsi9"><span>CRISPR technology</span></a><span> is transforming how we manipulate genes to understand their function. To explain CRISPR&#x2019;s emerging role in genetic engineering, let&#x2019;s use a metaphor from the world of software engineering. Suppose that a genome is like a massive codebase. There&#x2019;s a bug somewhere in the codebase, and you need to figure out which line is the culprit. Until CRISPR, altering a particular line of code has been laborious, expensive, and error-prone. CRISPR gives researchers the ability to &#x201C;comment out&#x201D;&#xA0;a particular line of code with pinpoint accuracy.</span></p>

<p><span>Furthermore, imagine if you could generate thousands of copies of your program, comment out a different line of code in each copy, and then compile and run all of the copies in parallel to determine which copies&#xA0;are error-free. <em>This hypothetical debugging is the essence of a CRISPR screen</em>. It allows scientists to test the function of thousands of genes in a single, parallelized experiment.</span></p>
<p><span>Together with the accompanying video, I&#x2019;ll explain how to start from a list of gene targets and finish with a ready-to-order <strong>library of oligos</strong> for your targeted CRISPR screen.</span></p>
<h2><span>Designing a targeted CRISPR screen</span></h2>
<p><span>CRISPR experiments have typically fallen into two camps: altering a single gene, or altering all</span><span> genes&#xB9; with a genome-wide&#xB2; screen</span><span>. The happy medium between these extremes is the <strong>targeted CRISPR screen</strong>, the focus of this article. A targeted CRISPR screen tests tens to thousands of genes at a time, often in a single flask, avoiding the cost and complexity of a genome-wide experiment. I have personally designed and executed more than a dozen targeted CRISPR screens. What follows are my recommendations for designing a targeted CRISPR screen. There are three steps:</span></p>
<ol>
<li><span>Choose your genes of interest</span></li>
<li><span>Use </span><a href="http://goo.gl/9t6rUz"><span>Quilt</span></a><span> to design cloning-ready oligos for your gRNA library</span></li>
<li><span>Order your oligos from the appropriate source (I use </span><a href="http://goo.gl/paVNkX"><span>Sigma</span></a><span> and </span><a href="http://goo.gl/ImfyqX"><span>CustomArray</span></a><span>)</span></li>
</ol>
<p><span>The number of genes in your list (step 1), times the number of gRNAs per gene, dictates the total number of gRNAs in your screen and, in turn, how you design, order, and clone your gRNA library&#xB3;</span><span>. By way of example, suppose I wish to design a new therapy that targets G-Protein Coupled Receptors (GPCRs)<sup>4</sup><span>. Drawing from publicly available genome-wide CRISPR screen designs, I would select 6 gRNAs per gene (the number present in the Zhang Lab GeCKOv2 library), yielding thousands of gRNAs for my library. Therefore I would choose a chip-based cloning technique to create my targeted CRISPR screen.</span></span></p>
<p class="jetpack-video-wrapper"><span class="embed-youtube"><iframe class="youtube-player" type="text/html" width="730" height="441" src="http://www.youtube.com/embed/B04h_OtNy3M?version=3&amp;rel=1&amp;fs=1&amp;autohide=2&amp;showsearch=0&amp;showinfo=1&amp;iv_load_policy=1&amp;wmode=transparent"></iframe></span></p>
<p><span>The above video demonstrates the CRISPR design process. The steps shown in the video are as follows:</span></p>
<ol>
<li><span>Drag-and-drop my gene list into </span><a href="https://goo.gl/PIO6gk">Quilt</a><span>. Click on my gene list.</span></li>
<li><span>Click Quilt &gt; Match, for <strong>one-to-many</strong> matching (since there are many gRNAs for each gene)</span></li>
<li><span>Select a &#x201C;CRISPR Design&#x201D; dataset as the right-hand dataset</span></li>
<li><span>The quilted dataset you generated will have the gRNA oligos ready to be ordered for making your targeted CRISPR screen.</span></li>
</ol>
<p><span>The CRISPR Design Datasets in Quilt are derived from genome-wide gRNA libraries from the leading CRISPR research labs in the world: the Lander Lab, Sabatini Lab, Weissman Lab, Yusa Lab, and Zhang Lab. Each lab has published papers on these screens, and </span><a href="https://goo.gl/oyuXmG"><span>open-sourced their CRISPR designs</span></a><span>. To determine which CRISPR system is best for your application, see </span><a href="http://goo.gl/nKRRHU"><span>Joel McDade&#x2019;s blog post on the topic</span></a><span>.</span></p>
<p><span>Quilt-Matching my genes of interest to a CRISPR Design dataset provides me with a list of oligos that are ready to be ordered. These oligos can then be cloned into my lentiviral vector to generate a library of gRNA-containing viruses. See the Appendix for the molecular biology details to help you get started with that process.</span></p>
<h2><span>Closing</span></h2>
<p><span>I hope that this article serves as a helpful guide as you get started with CRISPR screening. In future articles I plan to cover topics such as cloning libraries, making lentivirus, and analyzing screen results.</span></p>
<p><span>If you have any questions or topic requests, just post them to the comments.</span></p>
<p>Good luck, and keep going.</p>
<h2><span>Appendix: Basic molecular biology of gRNA design and cloning</span></h2>
<p><img class="alignnone size-full wp-image-31" src="https://quiltdata.files.wordpress.com/2016/01/schematic-1024.png?w=730" alt="schematic-1024"></p>
<h2><span>Plate-based cloning (hundreds of gRNAs)</span></h2>
<p><span>If you are designing a screen with hundreds of gRNAs, I recommend ordering 24bp forward and reverse oligos, pre-mixed in a 96-well plate. (I&#x2019;ve used Sigma&#x2019;s </span><a href="http://goo.gl/paVNkX"><span>pre-mix oligo plates</span></a><span> at the 100uM synthesis scale for this). You can use </span><a href="http://goo.gl/m0tTUI"><span>annealed oligo cloning</span></a><span> to anneal each forward/reverse oligo premix separately in plates. The next step is to pool all the annealed products together for a single ligation reaction into your cut lentiviral vector backbone.&#xA0;<strong>Important note</strong>: The sequence of the oligo overhangs (green in the diagram) depends on the the lentiviral vector that you choose. These overhangs are specific for the lentiviral vector found </span><a href="http://goo.gl/QufdWz"><span>here</span></a><span>, which is cut by the restriction enzyme BfuAI.</span></p>
<h2><span>Chip-based cloning (thousands of gRNAs)</span></h2>
<p><span>For anything more than a few hundred gRNAs, it&#x2019;s best to use a chip-based approach and order a pool of 74mer oligos. I use </span><a href="http://goo.gl/ImfyqX"><span>CustomArray</span></a><span>&#x2019;s oligo pools at both the 12K and 92K oligo scale. Oligo pools can be PCR amplified with primers ArrayF and ArrayR<sup>5</sup></span><span>. PCR amplification achieves three things: a double-stranded product, longer overhangs, and increased oligo concentration. After PCR, you can use </span><a href="http://goo.gl/mA2JHx"><span>Gibson Assembly</span></a><span> to clone into your cut vector.</span></p>
<h2>Footnotes</h2>
<ol>
<li>No screen really covers all genes, as the total gene count&#x2014;and even the very definition of &#x201C;gene&#x201D;&#x2014;changes frequently.</li>
<li><span>There are </span><a href="http://goo.gl/3Ib6YH"><span>multiple tools</span></a><span> for designing individual CRISPR guide RNAs (gRNAs), and there are </span><a href="http://goo.gl/oyuXmG"><span>publicly available</span></a><span>, pre-cloned genome-wide gRNA libraries.</span></li>
<li><span>If you have hundreds of total gRNAs, you&#x2019;ll use plate-based cloning. Whereas if you have thousands of gRNAs, you&#x2019;ll use chip-based cloning. See the Appendix for further details.</span></li>
<li><span>Two out of the top five selling drugs in the US, Hydrocodone and Lisinopril, target GPCRs#, and there are 769 GPCR genes in the human genome according to the </span><a href="http://goo.gl/RvmJP0"><span>Human Protein Atlas</span></a><span>.</span></li>
<li><span>The supplemental data of </span><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4089965/"><span>Shalem et al.</span></a><span> treats ArrayF and ArrayR in detail.</span></li>
</ol>
<div id="jp-post-flair" class="sharedaddy sd-like-enabled sd-sharing-enabled"><div class="sharedaddy sd-block sd-like jetpack-likes-widget-wrapper jetpack-likes-widget-unloaded" id="like-post-wrapper-104384596-19-56a77d0eaf6e1"><h3 class="sd-title">Like this:</h3><p class="likes-widget-placeholder post-likes-widget-placeholder"><span class="button"><span>Like</span></span> <span class="loading">Loading...</span></p><a class="sd-link-color"></a></div></div>			</div>

</div>
</body></html>
