# Age: It's All in Your Blood

[Original URL](http://sage.buckinstitute.org/age-its-all-in-your-blood/)

> We have all heard those particularly haunting tales about witches remaining ever youthful by imbibing a young woman's blood, but until a few years ago these tales were only told to frighten...

We have all heard those particularly haunting tales about witches remaining ever youthful by imbibing a young woman's blood, but until a few years ago these tales were only told to frighten children before bed. Last year, SAGE reported on a [study](http://sage.buckinstitute.org/frankenstein-parabiotic-mice/) where the blood of a young mouse was sufficient to rejuvenate an older mouse. This study lent credence to the idea that there must be something substantially different in young blood compared to old. To examine the changes that occur in blood as an individual ages, Dr. Andrew Johnson's[lab,](http://www.nhlbi.nih.gov/research/intramural/researchers/pi/johnson-andrew) at NIH/NHLBI (National Institute of Health/National Heart, Lung and Blood Institute) conducted an extensive study using thousands of patient blood samples, the study was then replicated, further verifying the results. The researchers chose to analyze the blood samples transcriptome, a measurement of the RNA transcripts from each gene. The compilation of RNA transcripts is a reflection of the relative expression levels of the genome at a given point in time. The choice to examine the transcriptome was pivotal, as all the cells in an organism will have the same DNA and this DNA does not generally change during the person's lifetime, thus making DNA genomic analysis less useful for an age-related study. What does change over a person's lifetime is modifications of DNA, which genes are expressed from the DNA and the relative levels of expression of each gene.

[](http://sage.buckinstitute.org/wp-content/uploads/2015/11/Blood-transcriptome-Blog-Illustration.jpg) Abstract of study showing blood samples taken across multiple ethnic groups, samples were then processed for RNA. The RNA from each person's blood sample was then analyzed for gene expression levels. Image created by Barbara J. Bailus

The study, which has been published in [Nature Communications](http://www.nature.com/ncomms/2015/151022/ncomms9570/full/ncomms9570.html), used certain types of blood cells and brain tissue to examine the age-associated changes in gene expression. In a remarkable show of replication, the study was initially performed with blood samples from individuals of European ancestry and then replicated in additional European ancestry samples, totaling an amazing 14,983 individual European ancestry samples. The study was then extended to various ethnic groups, including samples from individuals of Hispanic, African, or Native American ancestry. The study identified 1,497 genes in blood cells and/or brain tissue that showed significantly differential expression patterns in older individuals when compared to younger individuals.

The expression of the gene can either be negatively correlated (expressed at a lower level) or positively correlated (expressed at a higher level) in relation to chronological age. There were three distinct groups of genes that were negatively correlated with chronological age. The first group included three subgroups: ribosomal genes (factories on which a RNA is translated into a protein), mitochondrial genes (energy factories of the cells), and genes associated with DNA replication and repair (DNA maintenance and fidelity). All of the genes associated with these subgroups are vitally important to the health of a cell and tissue. The second large group consisted of genes associated with immunity. The third large group was composed of genes that code for the actual ribosomal subunits. Decreased gene expression could help explain the decreased "health" of older cells and increased mutation rates in older cells. There were also four groups of genes positively correlated with age, which were focused on cellular structure, immunity, fatty acid metabolism, and lysosome activity. Several of the genes in these clusters had been previously identified in other age-related screens in various model organisms, further supporting this study's methods and findings.

Another interesting finding in this study involved epigenetic patterns, specifically methylation on cytosines (one of the four nucleotide bases in DNA) and the predictive. Epigenetics can be thought of as the "grammar" of DNA, as it doesn't change the underlying pattern of DNA base pairs, but rather instructs how a gene is to be expressed. Methylation on cytosines is an epigenetic mark that can have a regulatory effect on how or if a gene is expressed. Methylation patterns are also dynamic, meaning that this pattern can change over time. This study showed that those genes whose expression pattern changed with age were highly enriched for the presence of regulatory cytosines. This could indicate how gene expression is controlled as the individual ages. There are several targeted methylation therapies in development that might potentially offer the ability to effectively and safely alter these methylation patterns for therapeutic purposes. The authors found that by combining the transcriptomic expression patterns and the epigenetic patterns a "chronological" age predictor could be used to better understand an individual's "age" in terms of health. Further refinement is needed, but this type of predictor could have a substantial impact on prediction, diagnosis and treatment of individuals, perhaps even allowing for preventive treatments before symptoms progress to disease level changes.

The sheer magnitude of this study, from the number of samples to the ethnic diversity of the participants, makes it a pioneer in the rapidly expanding field of transcriptomics. Until a few years ago, the methods and budgets did not exist for such a study to take place, but as the technology continues to increase, costs decline and more data will become available, enhancing our understanding of aging and allowing for us to better cope with age-associated changes. The 1,497 genes identified as being associated with chronological age offer a plethora of new targets from which we can better understand the aging process and age-related diseases. With the current progress being made in the gene therapy and drug fields it is possible that some of these 1,497 genes could potentially be manipulated to ameliorate many age-related diseases.

Dr. Andrew Johnson was able to answer a few questions for SAGE about this transformative study and how the aging field may benefit from the information generated:

[![Lab Photo - Andrew](http://sage.buckinstitute.org/wp-content/uploads/2015/11/Lab-Photo-Andrew.jpg)](http://sage.buckinstitute.org/wp-content/uploads/2015/11/Lab-Photo-Andrew.jpg) Dr. Andrew Johnson. Image credit Dr. Andrew Johnson.

**Q: What was the most challenging aspect of this study?**

**AJ:** The challenging aspects of leading the study from my standpoint were largely organizational. Given so many analyses and people involved it took a lot of persistence and organization to keep things on track, and maintain a vision for how things will come together rationally. An additional challenge was how to summarize and present significant results for ~1,500 genes. Most of us are more used to a situation where we discuss one or a handful of genes in our results. From a scientific standpoint we collectively had to answer some difficult questions about the best analysis approaches for combining our various data and doing mediation and prediction modeling.

**Q: How long did the actual data analysis take?**

**AJ:** This study took more than 4 years from the initial analysis plan to publication making it one of the longer ones I have been involved in. It started out a simple concept but it snow-balled a lot over time as we added more human cohorts and decided we wanted to add replication samples, co-expression network analyses and other cell types and tissues. Along the way publications came out regarding the methylation-aging clock and given that some cohorts also had methylation data this motivated us to pursue new questions like assessing the transcriptome as an aging predictor versus methylation, and doing deeper annotation of the results. All of the new analyses added up to more time and more "hands in the kitchen". This made the project a more exciting collaboration and that is why you see so many co-authors that made key contributions to different aspects of the study.

**Q: Will you be doing proteomics studies to see if the transcription correlates with the actual protein levels?**

**AJ:** That's an excellent question. There are lots of reasons we might want to pursue that including starting to determine which of these genes may be good druggable targets for aging-related conditions. There is plenty of evidence that RNA and protein often have non-linear relationships. So we would expect only a subset of RNAs and proteins to show similar significant patterns with respect to aging. We do not have plans to do this type of work on the same scale as the RNA studies. One of the main reasons is that proteomics approaches are less standardized and many of the same large cohort studies have not yet taken on large-scale proteomics so the data is not readily available in large samples at this point.

**Q: What is the next step for your laboratory in age-related research?**

**AJ:** One of the exciting aspects of this study that did not reach this initial publication was that we began partnering with model organism researchers to knock down orthologs for these novel human aging genes and examine effects on longevity and other phenotypes. We expect those studies that are well along now will result in several additional manuscripts further elucidating aging roles for some of the genes we discovered. Our study was mostly limited to a blood-centric view of aging and gene expression. We included analysis of normal, post-mortem brain samples from the NABEC/UKBEC consortia. We only found evidence for consistent age-expression associations in brain for about 25% of the blood-validate genes. This could be a factor of different tissues having different biology and also of more limited statistical power given the set of available brain samples was much smaller than the blood samples. So it is an open and interesting question how transferable age-expression signatures really are across tissues and cell types. We are delving into this a bit further for brain in particular. Finally, we are interested in and participating in some efforts to try to gain more insight via systems biology integration of different "omics" data with respect to aging. For example, if we combine information from genetic studies such as GWAS for human longevity or older age traits like cognitive decline or walking speed with information from our RNA studies, or other data types such as epigenetic modifications, are we able to pinpoint genes or regions that may be key nodes in healthy or abnormal aging?

**Q: What future applications do you foresee you see coming from this study?**

**AJ:** I view this study as a good starting point for providing a recipe list of aging genes for what I hope are many detailed mechanistic studies in humans and other model systems. Of course, it is not the first study to tackle this type of question and there is a lot of value in the other approaches such as surveying extreme-long lived individuals and studying organisms like yeast, nematodes, fruit flies and rodents. But this study is the largest one yet and included replication in the study design. So I think it does provide a strong set of results. We hope some of the identified genes and pathways could turn out to be intervention points.

A second area of possible applications is in helping push along prediction models for accelerated biological aging. We added the dynamic of the transcriptome to the epigenetic-aging clock. In years to come there may be additional biomarkers, proteomics and other approaches that could improve the accuracy and relevance of such predictions. There may be some forensic applications such as trying to estimate the age of an unknown John/Jane Doe blood sample where you can recover RNA. However, mainly the hope would be this type of information could be combined to cheaply determine relatively early in life when someone begins to deviate down a path of "rapid aging". If so, perhaps corrective actions such as healthier lifestyle choices could be strongly promoted to slow or prevent the development of age-related disease.

A third application of this study that may be non-obvious, but I think is important, is identifying confounding by age in RNA studies. For example, suppose you had a blood differential RNA study of n=20 drug responders versus n=20 non-responders. You may have a number of genes that are "significantly" different in RNA levels. Even if you have "age-matched" the groups to the point where the mean ages are "non-significant", if you have an average difference in age of only 1 or 2 years between your responder groups, some of the genes we identified in our study are so strongly associated with age they may confound your "drug response" results. Thus, as a manuscript reviewer or researcher for these types of studies, even if I tried to do age-matching, I would be asking questions like "Are any of these RNAs strongly associated with age in prior large RNA studies?" and "What happens to my results when I adjust for age as a covariate?"

**Q: What do you consider the most interesting or surprising aspect of this study?**

**AJ:** Both the most interesting and surprising aspect of the study was the extent in number and strength of the results. I did expect that since aging is a complex, multi-system process we would have tens and perhaps hundreds of significant results. I would not have predicted another order of magnitude greater set of results, many of which are incredibly consistent across studies and populations and reach tiny p-values (as low as 10^-577 !) To me this project is a good example of the power that can be achieved when a large set of scientists from around the world commit to working together.
